Phonebook/phonebook/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for phonebook.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# General Variables
+host: &host 'demo-phonebook.me'
+contactUrl: 'https://github.com/T-Systems-MMS/phonebook/issues'
+roomPlanningToolUrl: 'undefined'
+contactEmail: &contactEmail 'daniel-habenicht@outlook.de'
+ravenUrl: 'undefined'
+employeePictureEndpoint: 'undefined'
+assetsEndpoint: 'https://demo-phonebook.me/assets'
+
+# Frontend Variables
+frontend:
+  name: frontend
+  image:
+    repository: tsystemsmms
+    name: phonebook-frontend
+    tag: 1.22.0
+    pullPolicy: Always
+  replicaCount: 3
+
+
+nameOverride: ''
+fullnameOverride: ''
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations:
+    {}
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+   cpu: 100m
+   memory: 128Mi
+  requests:
+   cpu: 100m
+   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# Traefik Configuration 
+traefik:
+  startupArguments:
+    - ''
+  dashboard:
+    enabled: true
+    domain: board.demo-phonebook.me
+  ssl:
+    enabled: true
+    enforced: true
+  acme:
+    staging: false
+    enabled: true
+    email: *contactEmail
+    onHostRule: true
+    challengeType: 'http-01'
```

Comment:
is this the http specification we support? maybe we want support http2 also?

Reply:
nope. That is just the challenge format for traefik.